<launch>
  <include file="$(find place_recog_eval)/launch/base.launch" />

  <node pkg="place_recog_eval" type="evaluate.py" name="evaluate_node" output="screen" cwd="node"/>
</launch>
